pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -47,12 +47,9 @@ func defaultPredicates() sets.String {
 		predicates.MatchInterPodAffinityPred,
 		predicates.NoDiskConflictPred,
 		predicates.GeneralPred,
-		predicates.CheckNodeMemoryPressurePred,
```

Comment:
my approval doesn't cover this.  Commenting so the scheduling guys notice.

Reply:
yes, those are all replaced by the new feature.